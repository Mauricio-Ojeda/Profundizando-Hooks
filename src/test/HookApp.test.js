import { shallow } from "enzyme";
import HookApp from '../HookApp';

describe('Pruebas en HookApp.js', () => {
    test('Debe hacer un snapshot de HookApp', () => {
        const wrapper = shallow(<HookApp />)
        expect( wrapper ).toMatchSnapshot()
    });
})
