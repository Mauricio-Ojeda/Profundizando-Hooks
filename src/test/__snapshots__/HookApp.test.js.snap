// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HookApp.js Debe hacer un snapshot de HookApp 1`] = `
<div>
  <h1>
    Hola Hook App
  </h1>
</div>
`;
